src/com/flowers/FlowerSaver.java

```diff
@@ -0,0 +1,42 @@
+package com.flowers;
+
+import java.io.FileWriter;
+
+public final class FlowerSaver {
+
+    public static void save(String pathFile, Flower[] flowers) {
+        try {
+            FileWriter writer = new FileWriter(pathFile);
+            int rose = 0, chamomile = 0, tulip = 0;
```

Comment:
It is not a java style!!!
https://mate-academy.github.io/style-guides/java/java.html#s4.8.2-variable-declarations